src/wazuh_db/main.c

```diff
@@ -90,6 +93,14 @@ int main(int argc, char ** argv)
     wconfig.open_db_limit = getDefine_Int("wazuh_db", "open_db_limit", 1, 4096);
     nofile = getDefine_Int("wazuh_db", "rlimit_nofile", 1024, 1048576);
 
+    // Allocatting memory for backup configuration nodes
```

Comment:
Following the method name, this could be used for allocating future non-backup configs.
This comment should reflect this, or it can be directly removed.

Reply:
The comment was updated !